content/posts/irods-development-update-september-2024.md

```diff
@@ -0,0 +1,253 @@
+
+- [S3 Working Group](https://github.com/irods-contrib/irods_working_group_s3)
+
+    - have released C++ S3 API
+    - working meeting
+        - Fri, October 4, 2024, 3pm ET
+
+- [Metadata Templates Working Group](https://github.com/irods-contrib/irods_working_group_metadata_templates)
+
+    - draft paper for circulation among group
+    - server-side validation now working
+    - working meeting
+        - Tues, September 17, 2024, 10am ET
+
+- [TRiRODS](https://irods.org/trirods)
+
+    - Wed, December 4, 2024, 4pm ET
+    - speaker, TBD
+
+- [4.3.4](https://github.com/irods/irods/milestone/44)
+
+    - 61 bugs / 197 open, 1 closed
+
+- [4.3.5](https://github.com/irods/irods/milestone/43)
+
+    - 62 bugs / 216 open, 0 closed
+
+- [4.3 Backlog](https://github.com/irods/irods/milestone/34)
+
+    - 126 bugs / 297 open, 0 closed
+
+- [5.0.0](https://github.com/irods/irods/milestone/41)
+
+    - 6 bugs / 47 open, 2 closed
+
+- [5.0 Backlog](https://github.com/irods/irods/milestone/6)
+
+    - 0 bugs / 41 open, 0 closed
+
+- New Development Work
+
+    - working on new C++ project template for auth plugin
+
+- Active Development Work
+
+    - [iRODS Server](https://github.com/irods/irods)
+        - released 4.3.3
+        - working on C++ project template for database plugin
+        - working on refactor for iRODS 5.0.0
+            - simpler process model (configs only loaded at startup)
+            - hostname caching/config in catalog
+            - aiming for more cloud-friendly
+
+    - Build and Packaging
+        - working on moving more dependencies over to find_package
+
+    - iRODS Documentation
+        - added data_object_finalize database PEP implementation to Policy Cookbook
+        - added Testing section to Developer docs for how to add and run tests
+        - documented GenQuery2
+
+    - [PAM Interactive Authentication Plugin](https://github.com/irods/irods_auth_plugin_pam_interactive)
+        - released 0.1.0
+        - removed .irodsA.json feature
+        - added configuration for PAM stack
+        - fixed issues with authenticating while connected to catalog service consumer
+
+    - [Audit AMQP Plugin](https://github.com/irods/irods_rule_engine_plugin_audit_amqp)
+        - released 4.3.3.0
+
+    - [Unified Storage Tiering Capability](https://github.com/irods/irods_capability_storage_tiering)
+        - released 4.3.3.0
+
+    - [S3 Resource Plugin](https://github.com/irods/irods_resource_plugin_s3)
+        - released 4.3.3.0
+
+    - [Indexing Capability](https://github.com/irods/irods_capability_indexing)
+        - released 4.3.3.0
+
+    - [Logical Quotas](https://github.com/irods/irods_rule_engine_plugin_logical_quotas)
+        - released 4.3.3.0
+
+    - [Metadata Guard](https://github.com/irods/irods_rule_engine_plugin_metadata_guard)
+        - released 4.3.3.0
+
+    - [CURL microservices](https://github.com/irods/irods_microservice_plugins_curl)
+        - released 4.3.3.0
```

Comment:
note... many of these might move to 'background' for next month.

Reply:
True. It's not clear to me why that matters?